embed/script.js

```diff
@@ -0,0 +1,252 @@
+function h() {
+  var type = arguments[0] || 'div';
+  var attributes = arguments[1] || {};
+  var children = Array.prototype.slice.call(arguments, 2);
+  var el = document.createElement(type);
+
+  if (children[0] instanceof Array) {
+    children = children[0];
+  }
+
+  Object.keys(attributes)
+    .map(function(key) {
+      return [key, attributes[key]];
+    })
+    .forEach(function(data) {
+      var key = data[0],
+        value = data[1];
+      if (typeof value === 'function') {
+        el.addEventListener(
+          key,
+          function(e) {
+            value(e);
+          },
+          false
+        );
+      } else {
+        el.setAttribute(key, value);
+      }
+    });
+
+  if (children instanceof Array) {
```

Comment:
Just to smooth things out, we should maybe have our own type checking function e.g. https://gist.github.com/jhbsk/11083919

Maybe one for the future!

Reply:
Hanjee. Since this script would be loaded in external pages, idea is to keep it as lightweight as possible.